### PR TITLE
created new end point and tested for errors

### PR DESCRIPTION
**Pull request by Hammamsgit**
created patch end point to add or subtract article votes, and tested for errors accordingly

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -73,4 +73,48 @@ describe("app", () => {
...
+  describe("PATCH /api/articles/:article_id", () => {
+    test("status:200, responds with the updated article", () => {
+      const updateVote = { inc_votes: 1 };
+      return request(app)
+        .patch("/api/articles/3")
+        .send(updateVote)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.articles).toEqual({
+            article_id: 3,
```
Comment (daviidbartlett) [inline __tests__/app.test.js:85]: wouldn't expect this back on a key called article**s** consider renaming to the singular

File: __tests__/app.test.js [hunk 2]
```diff
@@ -73,4 +73,48 @@ describe("app", () => {
...
+          });
+        });
+    });
+    test("status: 404, responds with path not found", () => {
+      return request(app)
+        .patch("/api/deadend")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Route not found");
+        });
```
Comment (daviidbartlett) [inline __tests__/app.test.js]: don't need a path not found test for each endpoint!

File: __tests__/app.test.js [hunk 3]
```diff
@@ -73,4 +73,48 @@ describe("app", () => {
...
+    });
+    test("status: 400, responds with invalid request", () => {
+      return request(app)
+        .get("/api/articles/9999")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("BAD REQUEST");
+        });
```
Comment (daviidbartlett) [inline __tests__/app.test.js:102]: this is a 404, article not found

**Review by daviidbartlett — COMMENTED** [review]
You have some confusion in your tests. Missing a 404 article not found, but also you're doing get requests mid-way through the test suit 🤔

**Review by maddierosecodes — APPROVED** [review]
Comments addressed

**Pull request merged**